distribution/test/itests/test-itests-catalog/src/test/java/ddf/catalog/test/TestFederation.java

```diff
@@ -192,6 +192,18 @@ public void testFederatedQueryBySearchPhrase() throws Exception {
     }
 
     /**
+     * Tests Source can retrieve based on a pure spatial query
+     * @throws Exception
+     */
+    @Test
+    public void testFederatedSpatial() throws Exception {
+        String queryUrl = OPENSEARCH_PATH + "?lat=10.0&lon=30.0&radius=250000&spatialType=POINT_RADIUS" + "&format=xml&src="
+                + OPENSEARCH_SOURCE_ID;
+        when().get(queryUrl).then().log().all().assertThat()
+                .body(containsString(RECORD_TITLE_1), containsString(RECORD_TITLE_2));
```

Comment:
I would suggest adding a negative case of a record that should not be returned for that query.  That will protect against a regression that just returns all results.